src/components/Header/Header.styled.js

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+import {
+  bluePrince, white, 
+} from '../../utils/constant.styles';
+
+export const HeaderMain = styled.header`
```

Comment:
found the header is not adjusting quite right in the products list view
![image](https://user-images.githubusercontent.com/40035747/178130874-be86c52f-172b-41ea-b9b5-21d825791163.png)